gpustat/core.py

```diff
@@ -354,6 +373,7 @@ def get_process_info(nv_process):
                 'uuid': uuid,
                 'name': name,
                 'temperature.gpu': temperature,
+                'fan': fan,
```

Comment:
Let's make it be `fan.speed`. We are following the convention from `nvidia-smi --query-gpu`.

```
❯❯❯ nvidia-smi --help-query-gpu | grep fan
"fan.speed"
The fan speed value is the percent of maximum speed that the device's fan is currently intended to run at. It ranges from 0 to 100 %. Note: The reported speed is the intended fan speed. If the fan is physically blocked and unable to spin, this output will not match the actual fan speed. Many parts do not report fan speeds because they rely on cooling via fans in the surrounding enclosure.
```